index.js

```diff
@@ -1437,6 +1437,22 @@
     }
   };
 
+  //# not :: Boolean -> Boolean
+  //.
+  //. Takes a Boolean and returns the negation of that value
+  //. (`false` for `true`; `true` for `false`).
+  //.
+  //. ```javascript
+  //. > S.not(true)
+  //. false
+  //.
+  //. > S.not(false)
+  //. true
+  //. ```
+  S.not = def('not', [Boolean], function(x) {
+    return !x.valueOf();
+  });
```

Comment:
I believe `.valueOf()` is redundant here, since `!` will trigger a call to that method:

```javascript
> new Boolean(true)
new Boolean(true)

> !new Boolean(true)
false
```

The function can become `function(x) { return !x; }`, which is fairly nice even without arrow functions. :)